code/modules/projectiles/ammunition/ballistic/rifle.dm

```diff
@@ -119,7 +119,7 @@
 	ignore_throwspeed_threshold = TRUE
 	rip_time = 1.5 SECONDS
 	/// Amount of each type of damage healed per second
-	var/healing_per_second = 5
+	var/healing_per_second = -5
```

Comment:
Probably would be better to put this on line 146 with `var/healing = -healing_per_second * seconds_per_tick`, to keep the variable name true to its name.

Reply:
yeah if youre gonna call it HEALING per second the - implies it should be damaged

Reply:
does - variable even work i always did (variable * -1)

Reply:
it does